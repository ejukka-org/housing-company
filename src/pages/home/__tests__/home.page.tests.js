import React from 'react';
import HomePage from "../home.page";
import ReactDOM from "react-dom";
import { act } from "react-dom/test-utils";
import pretty from "pretty";
import { BrowserRouter } from "react-router-dom";

let container;

beforeEach(() => {
    container = document.createElement("div");
    document.body.appendChild(container);
});

afterEach(() => {
    document.body.removeChild(container);
    container = null;
});

test('Renders Home page', () => {
    act(() => {
        ReactDOM.render(
            <BrowserRouter>
                <HomePage />,
            </BrowserRouter>,
            container
        );
    });

    expect(pretty(container.innerHTML)).toMatchSnapshot();
});
