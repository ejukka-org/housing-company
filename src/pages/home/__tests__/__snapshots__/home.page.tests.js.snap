// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Home page 1`] = `
"<div class=\\"container\\">
  <div class=\\"container\\">
    <div class=\\"sc-bdVaJa eskUMO\\"><img style=\\"background-image: url();\\" class=\\"background-image\\">
      <div class=\\"sc-bwzfXH kXomTW\\">
        <h1 class=\\"sc-htpNat hgjgiu\\">MENU TITLE1</h1><span class=\\"sc-bxivhb fzOaMp\\">watch</span>
      </div>
    </div>
  </div>
  <div class=\\"container\\">
    <div class=\\"sc-bdVaJa eskUMO\\"><img style=\\"background-image: url();\\" class=\\"background-image\\">
      <div class=\\"sc-bwzfXH kXomTW\\">
        <h1 class=\\"sc-htpNat hgjgiu\\">MENU TITLE2</h1><span class=\\"sc-bxivhb fzOaMp\\">watch</span>
      </div>
    </div>
  </div>
</div>,"
`;
